doc/src/strip-metadata.md

```diff
@@ -0,0 +1,20 @@
+# `hevm strip-metadata`
```

Comment:
I am not sure functionality fits in with the rest of the things we do. I don't know the history for `hevm` well enough, but it may be an odd functionality to have?

Reply:
these are legacy hangovers from the dapptools days. I think we can probably remove them.